bom/pom.xml

```diff
@@ -28,9 +28,10 @@
   <packaging>pom</packaging>
 
   <properties>
+    <dropwizard.version>1.3.10</dropwizard.version>
```

Comment:
Note that DW 1.3.10 as is depends on Jetty 9.4.17 which suffers from https://github.com/eclipse/jetty.project/issues/3550#issuecomment-486153646

Reply:
@bentmann thx, looks like they are preparing 1.3.11 to resolve